lib/src/utils.dart

```diff
@@ -49,6 +49,13 @@ extension LogicValueMajority on LogicValue {
     }
     return result;
   }
+
+  /// Return the populationCount of 1s in a LogicValue
+  int popCount() {
```

Comment:
I'm a little concerned about continuing to grow these extensions as public API additions from ROHD-HCL.  If we add them to ROHD later, these will conflict, and we'll have some annoying decisions to make on deprecation and backwards incompatibility.  Can we make these (at least new) extensions private to ROHD-HCL, or even to the files that are using them?  I only see this one used in one test.

Reply:
Agreed.  I'll make these private.  Some are experimental and then end up being useful in a single case.  We can decide later what the best way to support these is if they end up being useful more broadly.